packages/searchbox/tests/site/index.html

```diff
@@ -0,0 +1,378 @@
+					<div class="au-search__btn">
+						<button class="au-btn au-btn--dark" type="submit"><span class="au-search__submit-btn-text">Search</span></button>
+					</div>
+				</form>
+
+				<h2>Responsive</h2>
+
+			<form role="search" class="au-search au-search--dark au-search--responsive au-form-group">
+				<label for="resp-dark" class="au-search__label">Search</label>
+				<input type="search" id="resp-dark" class="au-text-input au-text-input--dark" />
+				<div class="au-search__btn">
+					<button class="au-btn au-btn--dark"><span class="au-search__submit-btn-text">Search</span></button>
+				</div>
+			</form>
+
+			<h2>Icon only</h2>
+			<form role="search" class="au-search au-search--dark au-search--icon au-form-group">
+				<label for="icon-dark" class="au-search__label">Search</label>
+				<input type="search" id="icon-dark" class="au-text-input au-text-input--dark" />
+				<div class="au-search__btn">
+					<button class="au-btn au-btn--dark"><span class="au-search__submit-btn-text">Search</span></button>
+				</div>
+			</form>
+
+			<br>
+			<h2>Different text and visible label</h2>
+			<label for="vis-dark" class="au-label au-label--dark">Enter a domain</label>
+			<form role="search" class="au-search au-search--dark au-form-group">
+				<input type="search" id="vis-dark" class="au-text-input au-text-input--dark" />
+				<div class="au-search__btn">
+					<button class="au-btn au-btn--dark"><span class="au-search__submit-btn-text">Check availability</span></button>
+				</div>
+			</form>
+
+
+		</div>
+	</div>
+
+	<br><br><br>
+
+	<!-- HEADER WITH SEARCH BOX PLAIN-->
+	<h1> AU header with search</h1>
+	<header class="au-grid au-header" role="banner">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-9">
+					<a class="au-header__brand" href="#">
+						<img class="au-header__brand-image" alt="Insert alternate text here" src="https://designsystem.gov.au/assets/img/placeholder/256x80.png">
+						<div class="au-header__text">
+							<h1 class="au-header__heading">Site title</h1>
+							<div class="au-header__subline">
+								Service description that could be a little longer
+							</div>
+						</div>
+					</a>
+				</div>
+				<div class="col-md-3">
+					<form role="search"  class="au-search">
+						<label for="standard-nav" class="au-search__label">Search</label>
+						<input type="search" id="standard-nav" name="standard" class="au-text-input" />
+						<div class="au-search__btn">
+							<button class="au-btn"><spaform role="search"lass="au-search__submit-btn-text">Search</span></button>
+						</div>
+					</form>
+
+				</div>
+			</div>
+		</div>
+	</header>
+	<nav class="au-main-nav" aria-label="main">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-12">
+					<div id="main-nav-default-3" class="au-main-nav__content">
+						<button
+							aria-controls="main-nav-default"
```

Comment:
Ensure these are controlling the right `main-nav` element.

Reply:
yep nice pickup